### PR TITLE
2.13 release note

### PR DESCRIPTION
**Pull request by jngz-es**
### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch-ml-common.release-notes-2.13.0.0.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+## Version 2.13.0.0 Release Notes
+
+Compatible with OpenSearch 2.13.0
+
+
+### Experimental Features
```
Comment (ylwu-amzn) [inline release-notes/opensearch-ml-common.release-notes-2.13.0.0.md:6]: These should not be experimental features

File: release-notes/opensearch-ml-common.release-notes-2.13.0.0.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+* Suppress removal AccessController in java.security has been deprecated and marked for removal ([#2195](https://github.com/opensearch-project/ml-commons/pull/2195))
+* add remote predict thread pool ([#2207](https://github.com/opensearch-project/ml-commons/pull/2207))
+
+### Infrastructure
+
+
+### Bug Fixes
+* fix error code when executing agent ([#2120](https://github.com/opensearch-project/ml-commons/pull/2120))
+* fix npe when executing agent with empty parameter ([#2145](https://github.com/opensearch-project/ml-commons/pull/2145))
+* fix cohere chat blueprint ([#2167](https://github.com/opensearch-project/ml-commons/pull/2167))
```
Comment (ylwu-amzn) [inline release-notes/opensearch-ml-common.release-notes-2.13.0.0.md:28]: This is not bug fix, move to `### Documentation` part

File: release-notes/opensearch-ml-common.release-notes-2.13.0.0.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+* fix delete model cache on macOS causing model deploy fail with model ([#2180](https://github.com/opensearch-project/ml-commons/pull/2180))
+* adding BWC for connector config field ([#2184](https://github.com/opensearch-project/ml-commons/pull/2184))
+* Bump common-compress package to fix CVE ([#2186](https://github.com/opensearch-project/ml-commons/pull/2186))
```
Comment (ylwu-amzn) [inline release-notes/opensearch-ml-common.release-notes-2.13.0.0.md:31]: CVE is not bug fix. , move to `Maintenance` ?

File: release-notes/opensearch-ml-common.release-notes-2.13.0.0.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+* support question answering model ([#2208](https://github.com/opensearch-project/ml-commons/pull/2208))
+* Guardrails for remote model input and output ([#2209](https://github.com/opensearch-project/ml-commons/pull/2209))
+
+### Enhancements
+* Add verification to rate limiter number field ([#2113](https://github.com/opensearch-project/ml-commons/pull/2113))
+* Support Pretrained Model from on-prem or different cloud provider ([#2119](https://github.com/opensearch-project/ml-commons/pull/2119))
+* asymmetric embeddings ([#2123](https://github.com/opensearch-project/ml-commons/pull/2123))
+* Set the number of ml system index primary shards to 1 ([#2137](https://github.com/opensearch-project/ml-commons/pull/2137))
+* prevent exposing internal ip when an agent gets an internal OpenSearch exception ([#2154](https://github.com/opensearch-project/ml-commons/pull/2154))
+* Suppress removal AccessController in java.security has been deprecated and marked for removal ([#2195](https://github.com/opensearch-project/ml-commons/pull/2195))
```
Comment (ylwu-amzn) [inline release-notes/opensearch-ml-common.release-notes-2.13.0.0.md:19]: Move to `Maintenance` ?

File: release-notes/opensearch-ml-common.release-notes-2.13.0.0.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
+## Version 2.13.0.0 Release Notes
+
+Compatible with OpenSearch 2.13.0
+
+
+### Experimental Features
+* hidden agent ([#2204](https://github.com/opensearch-project/ml-commons/pull/2204))
+* Initiate MLInferencelngestProcessor ([#2205](https://github.com/opensearch-project/ml-commons/pull/2205))
```
Comment (ylwu-amzn) [inline release-notes/opensearch-ml-common.release-notes-2.13.0.0.md:8]: Why add this PR ?

File: release-notes/opensearch-ml-common.release-notes-2.13.0.0.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+* auto deployment for remote models ([#2206](https://github.com/opensearch-project/ml-commons/pull/2206))
+* support question answering model ([#2208](https://github.com/opensearch-project/ml-commons/pull/2208))
+* Guardrails for remote model input and output ([#2209](https://github.com/opensearch-project/ml-commons/pull/2209))
+
+### Enhancements
+* Add verification to rate limiter number field ([#2113](https://github.com/opensearch-project/ml-commons/pull/2113))
+* Support Pretrained Model from on-prem or different cloud provider ([#2119](https://github.com/opensearch-project/ml-commons/pull/2119))
```
Comment (ylwu-amzn) [inline release-notes/opensearch-ml-common.release-notes-2.13.0.0.md:15]: Why add this PR?

File: release-notes/opensearch-ml-common.release-notes-2.13.0.0.md (hunk 1 above)
Comment (jngz-es) [reply release-notes/opensearch-ml-common.release-notes-2.13.0.0.md:6]: OK

File: release-notes/opensearch-ml-common.release-notes-2.13.0.0.md (hunk 2 above)
Comment (jngz-es) [reply release-notes/opensearch-ml-common.release-notes-2.13.0.0.md:28]: done

File: release-notes/opensearch-ml-common.release-notes-2.13.0.0.md (hunk 3 above)
Comment (jngz-es) [reply release-notes/opensearch-ml-common.release-notes-2.13.0.0.md:31]: done

File: release-notes/opensearch-ml-common.release-notes-2.13.0.0.md (hunk 4 above)
Comment (jngz-es) [reply release-notes/opensearch-ml-common.release-notes-2.13.0.0.md:19]: done

File: release-notes/opensearch-ml-common.release-notes-2.13.0.0.md (hunk 5 above)
Comment (jngz-es) [reply release-notes/opensearch-ml-common.release-notes-2.13.0.0.md:8]: My bad, removed.

File: release-notes/opensearch-ml-common.release-notes-2.13.0.0.md (hunk 6 above)
Comment (jngz-es) [reply release-notes/opensearch-ml-common.release-notes-2.13.0.0.md:15]: removed

**Comment by ylwu-amzn** [comment]
Please include all PRs merged to 2.13, I randomly check last two PRs merged to 2.13, they are missing.
1. Add integration tests for the RAG pipeline covering OpenAI and Bedrock
2. add local model enabling/disabling setting

**Comment by jngz-es** [comment]
Went through all new commits in 2.13 branch and added missing PRs.

**Comment by dhrubo-os** [comment]
we need to add this PR: https://github.com/opensearch-project/ml-commons/pull/2251 also

**Comment by jngz-es** [comment]
> we need to add this PR: #2251 also

It is a very new one. We expect more PRs coming, we can have it in next update pr. Otherwise we only have the release note on the last day which is currently blocking building team.

**Comment by jngz-es** [comment]
Added pr 2236.

**Review by dhrubo-os — APPROVED** [review]

**Review by b4sjoo — APPROVED** [review]

**Pull request merged**